tests/functional/lib/helpers.js

```diff
@@ -654,23 +654,24 @@ define([
   /**
    * Open FxA from the untrusted OAuth relier.
    *
-   * @param {object} context
    * @param {string} page - page to open
    * @param {object} [options]
    * @param {string} [options.header] - element selector that indicates
    *  "page is loaded". Defaults to `#fxa-force-auth-header`
    * @param {object} [options.query] - query strings to open page with
    */
-  function openFxaFromUntrustedRp(context, page, options) {
+  function openFxaFromUntrustedRp(page, options) {
     options = options || {};
     options.untrusted = true;
-    return openFxaFromRp(context, page, options);
+    return function () {
```

Comment:
Possibly I'm not thinking it through properly here, but can't we just return the function returned by `openFxaFromRp` directly, rather than wrapping this call in another function?